docs/advanced.md

```diff
@@ -38,3 +38,42 @@ The Dashboard panel should now show be showing those APIs:
 Once these steps have been completed, you will be able to use these credentials in the extension.
 In the `jupyterlab.google-drive` settings of the settings registry, set the **clientID** field to be the client id provided by the developer console. If everything is configured properly, you should be able to use the application with your new credentials.
 ![Client ID](images/clientid.png)
+
+### Seeding JupyterLab images with Google credentials
+While adding credentials via the settings functionality from within JupyterLab is possible, as described above, users may also wish to pre-seed these settings so the extension works out-of-the-box on start-up.
+
+With a [redesign](https://github.com/jupyterlab/jupyterlab/pull/2936) of JupyterLab's setting system, the location of settings pertaining to the google-drive extension changed.  When you manually add credentials from within JupyterLab, they now get persisted to:
```

Comment:
I don't think the redesign here is that relevant to new users, and this comment will go out of date fairly quickly. Perhaps we can just give the location for the file?